string faceId1
string faceId2
---
float32 result